quinn-proto/src/congestion/bbr/mod.rs

```diff
@@ -344,8 +345,8 @@ impl Bbr {
         if self.recovery_window >= bytes_lost {
             self.recovery_window -= bytes_lost;
         } else {
-            const K_MAX_SEGMENT_SIZE: u64 = MAX_DATAGRAM_SIZE;
-            self.recovery_window = K_MAX_SEGMENT_SIZE;
+            let k_max_segment_size: u64 = self.current_mtu;
```

Comment:
Style nit: the indirection through a named binding (with an unnecessary type annotation) seems pretty pointless, how about assigning directly and annotating with a comment to clarify the context here?

Reply:
Good catch, it looks like I forgot to clean up after replacing the constant by the current mtu